helm-charts/konk/scripts/provision.sh

```diff
@@ -1,6 +1,16 @@
 #!/bin/bash
 set -xe
 
+# load existing CA
```

Comment:
As part of #113, `/etc/kubernetes/pki/etcd/ca.crt` should also be loaded from an existing secret when possible.